style/gallery.less

```diff
@@ -84,13 +84,14 @@
 			background-position: center;
 
 			img {
-				visibility: hidden;
+				background-color: #fff;
 			}
 
 			&-caption {
 				position: absolute;
+				background-color: #222;
 				bottom: 35px;
-				width: calc( 100vw - 32px );
+				width: calc( 100vw + 1px );
```

Comment:
_very strange, right? but I found this magic number here helps in the ios safari browser._ what's your finding here?

Reply:
Duude... yes, very strange indeed. Took me a while to realize that was breaking the sliding for the caption, as in everything was sliding except the caption (which would remain the same from the first image rendered in full screen). This might change later when we work on https://phabricator.wikimedia.org/T262373. For the image info (caption/licensing) to be fixed on top of the image I needed the width to cover 100vw of the width, but it wouldn't work without `calc`

Reply:
Yes, we can check this again on another ticket.